src/YUILoader.cc

```diff
@@ -156,6 +157,35 @@ void YUILoader::loadUI( bool withThreads )
     }
 }
 
+void YUILoader::loadTestFrameworkPlugin( )
+{
+        // Do not try to login if test framework is not enabled
+        if( !getenv( YUITest_ENABLE )  && strcmp(getenv( YUITest_ENABLE ), "1") != 0 ) return;
```

Comment:
Move this to the other environment variables for consistency? Then the code here becomes a lot easier to read:

```C++
if ( strcmp( envTestEnable, "1" ) != "1" )
    return;
```

Reply:
duh...

```C++
if ( strcmp( envTestEnable, "1" ) != 0 )
    return;
```

of course.

Reply:
Idea was to reuse it in ncurses and qt plugins, but it's not yet the case, so I've change that. Thanks!